### DISCUSSION TITLE
Currencies quick question.

### DISCUSSION THREAD

**Discussion by holden** [swellstores/community#174] [Help] [answered]
I see currencies and locales both list in the headers menu as options in the swell storefront menu but they don't appear to be there in the origin theme by default?  We've enabled a few different currencies for our project and they show up during checkout but do not appear to be enabled in the origin theme. What else do we need to to do to be able to get this to show up?  Do we have to add it to the theme files manually ourselves as well as in the storefront menu?

Thanks,

**Comment by sags95** [comment]
@holden Would you be able to mention the Origin version you're running and the store id?

**Reply by holden** [reply]
Sure. Using the latest version 2.0.12 and our store id is `cirqus`.

Thanks

**Reply by holden** [reply]
Any ideas? Is there something I need to add to the template to make this work, as it seems as if it should work out of the box but just doesn't.  Unless there's some kind of configuration I have missed.

Please let me know, thanks.

**Reply by logeshswell** [reply, marked as answer]
@holden There was a configuration issue in your store and I fixed it now. So, you should see multi-currencies displayed in your origin storefront now.

**Comment by rodrigoddalmeida** [comment]
@holden could you update your storefront (Origin) to the latest version and confirm if the issue persists?

**Reply by holden** [reply]
Ok, I upgraded it to `2.0.14` however it didn't seem to make any difference. Except that translations also seem to be breaking #170 

https://cirqus--dev.swell.store/products/evenchilada-music-bingo/

Do I need to do something else in the settings for this to work?

**Reply by rodrigoddalmeida** [reply]
@holden could you try v2.0.15 - it fixed issues with language tokens caused by a dependency change.

**Reply by holden** [reply]
I upgraded to `v2.0.15` but it doesn't seem to make a difference.

**Comment by holden** [comment]
Also I noticed on the checkout page itself, when you change the currency, for example from USD to GBP, it doesn't correctly change the total. It updates the subtotal correctly but the total is just USD with the Pound sign which is completely wrong.

<img width="544" alt="CleanShot 2023-03-24 at 11 55 06@2x" src="https://user-images.githubusercontent.com/48241/227503447-abe9263b-1c04-4ae0-9935-764d12120b1f.png">

![CleanShot 2023-03-24 at 11 58 50@2x](https://user-images.githubusercontent.com/48241/227504284-99caa56c-0e84-4079-a316-01bb7d8e789d.png)

**Reply by steven-k1** [reply]
Hello @holden, We are aware of this issue and have it logged into our system to be worked on. As updates become available, we will let you know more.
